src/expandPath.ts

```diff
@@ -26,9 +21,7 @@ export function expandPath(path?: string, rootPath?: string): string | undefined
 		path = path.replace("{workspaceFolder}", <string>rootPath);
 	}
 
-	if (process.platform === "win32" && !path.startsWith("/")) {
-		path = "/mnt/" + path.substr(0, 1).toLowerCase() + path.substr("X:".length).split("\\").join("/");
-	}
+	path = "/mnt/" + path.substr(0, 1).toLowerCase() + path.substr("X:".length).split("\\").join("/");
```

Comment:
In extension.ts (selectProgramName, getProgramName), there is converting to WSL path in 2 places (using the rootPath optional arg which is available at this point):
```
.then(v => expandPath(v, vscode.workspace.rootPath)
```
So i think this if-check is needed here, otherwise second call in actual debugger will produce wrong path...

Am I correct?

Otherwise pull request seems fine as far as I can tell...

Reply:
What if we split expandPath into 2 methods - one to replace workspace dir, second one to change to WSL path?

Then in extension we use only replacing workspace dir, while in debugger we change to WSL...

Reply:
Wow, I overlooked `&& !path.startsWith("/")`!
I rewrote commit "Remove OS dependency from expandPath".
Is this okay?